scripts/install/autobrr.sh

```diff
@@ -0,0 +1,189 @@
+        bash /etc/swizzin/scripts/nginx/autobrr.sh
+        systemctl reload nginx
+        echo_progress_done
+    fi
+    exit 0
+fi
+
+_install_autobrr() {
+
+    echo_progress_start "Downloading release archive"
+
+    case "$(_os_arch)" in
+        "amd64") arch='x86_64' ;;
+        "arm64") arch="arm64" ;;
+        "armhf") arch="armv6" ;;
+        *)
+            echo_error "Arch not supported"
+            exit 1
+            ;;
+    esac
+
+    latest=$(curl -sL https://api.github.com/repos/autobrr/autobrr/releases/latest | grep "linux_$arch" | grep browser_download_url | cut -d \" -f4) || {
+        echo_error "Failed to query GitHub for latest version"
+        exit 1
+    }
+
+    if ! curl "$latest" -L -o "/tmp/autobrr.tar.gz" >> "$log" 2>&1; then
+        echo_error "Download failed, exiting"
+        exit 1
+    fi
+    echo_progress_done "Archive downloaded"
+
+    echo_progress_start "Extracting archive"
+
+    # the archive contains both autobrr and autobrrctl to easily setup the user
+    tar xfv "/tmp/autobrr.tar.gz" --directory /usr/bin/ >> "$log" 2>&1 || {
+        echo_error "Failed to extract"
+        exit 1
+    }
+    rm -rf "/tmp/autobrr.tar.gz"
+    echo_progress_done "Archive extracted"
+}
+
+_autobrr_user_config() {
+    echo_progress_start "Configuring autobrr"
+
+    # get random available port
+    port=$(port 10000 12000)
+
+    # generate a sessionSecret
+    sessionSecret="$(head /dev/urandom | tr -dc A-Za-z0-9 | head -c16)"
+
+    if [ ! -d "/home/$user/.config/autobrr/" ]; then
+        mkdir -p "/home/$user/.config/autobrr/"
+    fi
+    chown -R "$user": "/home/$user/.config/autobrr"
+
+    cat > "/home/$user/.config/autobrr/config.toml" << CFG
+# config.toml
+# Hostname / IP
+#
+# Default: "localhost"
+#
+host = "127.0.0.1"
+# Port
+#
+# Default: 8989
+#
+port = ${port}
+# Base url
+# Set custom baseUrl eg /autobrr/ to serve in subdirectory.
+# Not needed for subdomain, or by accessing with the :port directly.
+#
+# Optional
+#
+baseUrl = "/autobrr/"
+# autobrr logs file
+# If not defined, logs to stdout
+#
+# Optional
+#
+#logPath = "log/autobrr.log"
+# Log level
+#
+# Default: "DEBUG"
+#
+# Options: "ERROR", "DEBUG", "INFO", "WARN"
+#
+logLevel = "DEBUG"
+
+# Session secret
+#
+sessionSecret = "${sessionSecret}"
+CFG
+
+    chown -R "$user": "/home/$user/.config/autobrr"
+
+    echo_progress_done
+
+}
+
+_autobrr_add_user() {
+    # use autobrrctl to add user into database
+    echo_progress_start "Add user"
+
+    for user in "${users[@]}"; do
+        echo_log_only "Adding user $user"
+        pass=$(_get_user_password "$user")
+
+        # the password needs to be created with argon2 so we use autobrrctl to create the user
+        # using sqlite3 directly was not an option
+        echo -n "$pass" | autobrrctl --config "/home/$user/.config/autobrr" create-user "$user"
+    done
+}
+
+_systemd_autobrr() {
+    if [[ ! -f /etc/systemd/system/autobrr@.service ]]; then
```

Comment:
I'd just _always_ overwrite the systemd service if this is being installed. Otherwise some edge case could keep on re-using an out of date service file. i.e. I'd remove this `if fi`